Firmware/ultralcd.cpp

```diff
@@ -7232,6 +7240,17 @@ uint8_t get_message_level()
 {
 	return lcd_status_message_level;
 }
+/**/
```

Comment:
Did you plan to add some comments  here?

Reply:
You are right, I was trying something and forgot to remove my marks